frontend/lib/parsing/ParserContextImpl.h

```diff
@@ -2399,11 +2387,11 @@ ParserContext::buildVarOrMultiDeclStmt(YYLTYPE locEverything,
       if (elt->isComment()) delete elt;
     }
     delete vars;
-    if (!isBuildingTupleDecl) {
-      auto attributeGroup = buildAttributeGroup(locEverything);
-      if (attributeGroup) {
-        lastDecl->attachAttributeGroup(std::move(attributeGroup));
-      }
+    // for single element decls, we attach the attribute group to the decl
```

Comment:
One concern about this is that I think we had a goal to have the children be roughly parse order. I don't know how much we care about that right now, I'd ask Michael.

Reply:
It is not important to me that this PR preserve that property for Attributes. If we put the attribute group at the end of the children, we could note repairing it as Future Work.

*However* I think we have some uAST nodes that assume that they control the end of the list. If they do, we cannot ensure that the attribute group is first, such uAST nodes have to change in order to track both the beginning and end of the relevant kind of child nodes. I have some concern that we will not notice all such cases that need updating. (Of course, perhaps there aren't any uAST nodes that assume this anymore).

Reply:
`TupleDecl` was one of the nodes that assumed it owned the back of the list, but I have updated it to account for the `AttributeGroup` maybe being the last item instead of the first (it will never be the first if it was built from `ParserContextImpl`)